package.json

```diff
@@ -4,35 +4,37 @@
   "version": "0.4.2",
   "description": "An eslint plugin for better valtio experience",
   "main": "index.js",
+  "files": [
+    "**"
+  ],
   "scripts": {
+    "prepare": "husky install",
     "prebuild": "shx rm -rf dist",
     "build": "rollup -c",
     "postbuild": "yarn copy",
-    "eslint": "eslint --fix '{src,tests}/**/*.{js,ts,jsx,tsx}'",
-    "eslint:ci": "eslint '{src,tests}/**/*.{js,ts,jsx,tsx}'",
-    "prepare": "yarn build",
+    "eslint": "eslint --fix '{src,tests}/**/*.{js,jsx}'",
+    "eslint:ci": "eslint '{src,tests}/**/*.{js,jsx}'",
```

Comment:
Hoped to check `'*.{js,json}'` too but something is blocking?

Reply:
I wasn't able to fix this and nor was I able to find documentation regarding it. 
Eslint was trying to validate it's own config and `package.json` but then expecting semi-colon's in json files. Which didn't make sense, there's not much about this issue anywhere, I tried downgrading but no luck

Reply:
okay, let's leave it. thanks.